### ISSUE TITLE
Valid address and ip for testing taxes

### ISSUE THREAD

**Issue by alko89** [faker-js/faker#1206]
### Clear and concise description of the problem

Faker works great when you're trying to generate random address and ip locations, but the generated addresses are not good when you want to test valid locations for tax calculation.

### Suggested solution

Add a new method that would return a random address and ip card that would be valid for tax validation.

**Comment by xDivisionByZerox** [comment]
The first line of our website:

> Faker is a popular library that generates fake (but reasonable) data that can be used for things such as: [...]
 
Im not sure what you mean by "valid". 
Like real-world data "valid"? Well, that's not what we are aiming for. 
If you mean "valid" like logically current, then I would classify this as a `bug`.

**Comment by alko89** [comment]
Yeah I meant real world valid. Basically the address needs to be recognized as taxable in an accounting software, kind of like these ones:

https://www.prepostseo.com/tool/fake-address-generator

**Comment by ST-DDT** [comment]
**Team decision**

We won't add a database with real world data.
